cookbooks/tile/templates/default/export.erb

```diff
@@ -54,6 +55,115 @@ def output_error(message, status = "400 Bad Request"):
   print("</body>")
   print("</html>")
 
+# Add a copyright notice for raster formats (PNG, JPEG, WEBP)
+def add_copyright_notice_raster(image, map_width, map_height, format):
+  #  Create a temporary file for the PNG image
+  with tempfile.NamedTemporaryFile(suffix=".png") as png_tempfile:
+    # Save the Mapnik image into the temporary file as PNG
+    png = image.tostring("png")
+    png_tempfile.write(png)
+
+    # Load the PNG data from the temporary file into a Cairo ImageSurface
+    surface = cairo.ImageSurface.create_from_png(png_tempfile.name)
+
+  add_copyright_notice_vector(surface, map_width, map_height)
+
+  # Create a temporary file for the output image
+  with tempfile.NamedTemporaryFile(suffix="." + format) as img_tempfile:
+    # Convert the Cairo surface to PNG and save it to the temporary file
+    surface.write_to_png(img_tempfile.name)
+
+    if format == "png":
+      img_tempfile.seek(0)
+      return img_tempfile.read()
+    else:
+      # Open the output PNG image for conversion to other formats
+      img = Image.open(img_tempfile.name)
```

Comment:
I think `Image.frombytes` could be used here to ingest the backing data from the in-memory file?